src/uct/sm/scopy/cma/cma_ep.c

```diff
@@ -37,18 +37,23 @@ const struct {
 
 static UCS_CLASS_INIT_FUNC(uct_cma_ep_t, const uct_ep_params_t *params)
 {
-    UCT_CHECK_PARAM(params->field_mask & UCT_EP_PARAM_FIELD_IFACE_ADDR,
-                    "UCT_EP_PARAM_FIELD_IFACE_ADDR and UCT_EP_PARAM_FIELD_DEV_ADDR are not defined");
-
+    UCT_EP_PARAMS_CHECK_DEV_IFACE_ADDRS(params);
     UCS_CLASS_CALL_SUPER_INIT(uct_scopy_ep_t, params);
+
     self->remote_pid = *(const pid_t*)params->iface_addr &
                        ~UCT_CMA_IFACE_ADDR_FLAG_PID_NS;
+
+    self->keepalive = uct_ep_keepalive_create(self->remote_pid);
```

Comment:
maybe create it on-demand, when ep_check is called. like for mm_ep

Reply:
can do that. Should I update cuda_ipc as well? (i recall a discussion where we decided to initialize it always for cuda_ipc)

Reply:
if can be done easily, then yes